<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>docsView</class>
 <widget class="QDialog" name="docsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonExit">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelOwner">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Выберите владельца документов:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBoxUser">
           <property name="text">
            <string>Пользователь</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="checkBoxClient">
           <property name="text">
            <string>Клиент</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxClient"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBoxCase">
           <property name="text">
            <string>Дело</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCase"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTabWidget" name="widgetDocs">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitterFoto">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableView" name="viewFoto">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1280</height>
           </size>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>42</x>
             <y>200</y>
             <width>171</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Выберите действие</string>
           </property>
          </widget>
          <widget class="Line" name="line">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>231</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonAddFoto">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>230</y>
             <width>80</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
          <widget class="QLabel" name="labelPreview">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>231</width>
             <height>171</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonDelFoto">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>320</y>
             <width>81</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonShowIm">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>260</y>
             <width>80</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Просмотр</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonSaveFoto">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>350</y>
             <width>141</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Сохранить на ПК</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonSelectAllFoto">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>290</y>
             <width>101</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Выбрать все</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonScan">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>230</y>
             <width>101</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Сканировать</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonPrintFoto">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>260</y>
             <width>101</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Печать</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonRenameFoto">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>320</y>
             <width>121</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Переименовать</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitterAudio">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableView" name="viewAudio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1280</height>
           </size>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="Line" name="line_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>231</width>
             <height>20</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonAddAudio">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>190</y>
             <width>80</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>231</width>
             <height>161</height>
            </rect>
           </property>
           <property name="title">
            <string>Проигрыватель аудио</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <widget class="QPushButton" name="buttonPlayAudio">
            <property name="geometry">
             <rect>
              <x>9</x>
              <y>50</y>
              <width>91</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Play / Pause</string>
            </property>
           </widget>
           <widget class="QSlider" name="sliderAudioDur">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>85</y>
              <width>191</width>
              <height>21</height>
             </rect>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
            <property name="tickInterval">
             <number>10</number>
            </property>
           </widget>
           <widget class="QLabel" name="labelAudioSec">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>120</y>
              <width>58</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLCDNumber" name="lcdTime">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>32</y>
              <width>91</width>
              <height>41</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>120</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
           <widget class="QSlider" name="volumeSlider">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>140</y>
              <width>191</width>
              <height>16</height>
             </rect>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </widget>
          <widget class="QPushButton" name="buttonSelectAllAudio">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>220</y>
             <width>101</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Выбрать все</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonDelAudio">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>250</y>
             <width>80</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonSaveAudio">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>310</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Сохранить на ПК</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonRenameAudio">
           <property name="geometry">
            <rect>
             <x>64</x>
             <y>280</y>
             <width>141</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Переименовать</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Страница</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableView" name="viewText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>190</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>1280</height>
           </size>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="buttonNewText">
           <property name="geometry">
            <rect>
             <x>69</x>
             <y>40</y>
             <width>131</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Новый файл</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonEdittextDoc">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>130</y>
             <width>131</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Редактировать</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonAddText">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>70</y>
             <width>80</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonRenameText">
           <property name="geometry">
            <rect>
             <x>59</x>
             <y>100</y>
             <width>151</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Переименовать</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonSaveText">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>160</y>
             <width>131</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Сохранить на ПК</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonDeleteText">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>190</y>
             <width>80</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
          <widget class="QPushButton" name="buttonSelectAllText">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>220</y>
             <width>121</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Выбрать все</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
